build/release.mk

```diff
@@ -20,7 +20,7 @@ build-client:
 
 	cd $(BUILD_WEBAPP_DIR) && $(MAKE) build
 
-package:
+package: config-reset
```

Comment:
I'm worried about adding `config-reset` here -- is it strictly needed? The Jenkins build should already invoke `config-reset`, and I sometimes use `make package` to build a local package, but this would also have the unfortunate side effect of clobbering my testing configuration.

Could we instead replace the `cp -L config/config.json $(DIST_PATH)/config` with an invocation of your new tool to write the output directly to `$(DIST_PATH)/config`?